components/MessageCard.vue

```diff
@@ -14,49 +15,89 @@
 
 <script>
 export default {
-  props: ["message", "isThread"],
+  props: ["message", "isThread", "users"],
 
   computed: {
+    senderName() {
+      const usersDic = this.usersDisctionary(this.users);
```

Comment:
Dictionaryな変数名称はdictをキーワード名として共通化してください。
```suggestion
      const usersDict = this.usersDisctionary(this.users);
```